### PR TITLE
fix: Add retrieve VC retries

### PR DESCRIPTION
**Pull request by DRK3**
Signed-off-by: Derek Trider <Derek.Trider@securekey.com>

### CODE REVIEW

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -689,11 +689,27 @@ func (o *Operation) retrieveCredentialHandler(rw http.ResponseWriter, req *http.
...
 
-	docURLs, err := o.queryVault(profile, id)
-	if err != nil {
-		o.writeErrorResponse(rw, http.StatusInternalServerError, err.Error())
+	var docURLs []string
 
-		return
+	for retries := 0; retries < 10; retries++ {
+		if retries != 0 {
+			time.Sleep(time.Second * 5)
```
Comment (sandrask) [inline pkg/restapi/vc/operation/operations.go:696]: 5 seconds is too long in between first and second retry

Comment (troyronda) [reply pkg/restapi/vc/operation/operations.go:696]: @bstasyszyn

Comment (troyronda) [reply pkg/restapi/vc/operation/operations.go:696]: also 50 seconds maximum is too much :).

Comment (bstasyszyn) [reply pkg/restapi/vc/operation/operations.go:696]: For example, see: https://github.com/trustbloc/fabric-mod/blob/ba9a41234a5002930b1232c4640b86fbd1bf9bb7/common/util/retry/retry.go#L47

Comment (DRK3) [reply pkg/restapi/vc/operation/operations.go]: Reworked it - take a look and let me know.

File: pkg/restapi/vc/operation/operations.go [hunk 2]
```diff
@@ -693,11 +693,32 @@ func (o *Operation) retrieveCredentialHandler(rw http.ResponseWriter, req *http.
 		return
 	}
 
-	docURLs, err := o.queryVault(profile, id)
-	if err != nil {
-		o.writeErrorResponse(rw, http.StatusInternalServerError, err.Error())
+	var docURLs []string
 
-		return
+	backoff := 250 * time.Millisecond
```
Comment (bstasyszyn) [inline pkg/restapi/vc/operation/operations.go:698]: These values (initial backoff, maxRetries, and backoff factor) should be configurable.

Comment (fqutishat) [reply pkg/restapi/vc/operation/operations.go:698]: you should add retry logic outside operation as common library in edge-core

Comment (DRK3) [reply pkg/restapi/vc/operation/operations.go]: Done - and it's being used here now.

**Pull request merged**